content/en/observability_pipelines/guide/route_logs_in_datadog_rehydratable_format_to_Amazon_S3.md

```diff
@@ -0,0 +1,280 @@
+1. Add a query that filters out all logs going through log pipelines so that none of those logs go into this archive. For example, add the query `observability_pipelines_read_only_archive`, assuming no logs going through the pipeline have that tag added.
+1. Select **AWS S3**.
+1. Select the AWS Account that your bucket is in.
+1. Enter the name of the S3 bucket.
+1. Optionally, enter a path.
+1. Check the confirmation statement.
+1. Optionally, add tags and define the maximum scan size for rehydration. See [Advanced settings][5] for more information.
+1. Click **Save**.
+
+See the [Log Archives documentation][6] for additional information.
+
+## Configure the Datadog Archives destination
+
+You can configure the `datadog_archives` destination using the [configuration file](#configuration-file) or the [pipeline builder UI](#configuration-file).
+
+### Configuration file
+
+For manual deployments, the [sample pipelines configuration file][7] for Datadog includes a sink for sending logs to Amazon S3 under a Datadog-rehydratable format.
+
+{{< tabs >}}
+{{% tab "Docker" %}}
+
+In the sample pipelines configuration file, replace `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` with the AWS credentials you created earlier.
+
+{{% /tab %}}
+{{% tab "AWS EKS" %}}
+
+In the sample pipelines configuration file, replace `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` with the AWS credentials you created earlier.
+
+{{% /tab %}}
+{{% tab "APT-based Linux" %}}
+
+In the sample pipelines configuration file, replace `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` with the AWS credentials you created earlier.
+
+{{% /tab %}}
+{{% tab "RPM-based Linux" %}}
+
+In the sample pipelines configuration file, replace `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` with the AWS credentials you created earlier.
+
+{{% /tab %}}
+{{% tab "Terraform (AWS)" %}}
+
+Replace `${DD_ARCHIVES_BUCKET}` and $`{DD_ARCHIVES_REGION}` parameters based on your S3 configuration.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+<div class="alert alert-warning">If the Worker is ingesting logs that are not coming from the Datadog Agent and are routed to the Datadog Archives destination, those logs are not tagged with <a href="https://docs.datadoghq.com/logs/log_configuration/attributes_naming_convention/#reserved-attributes">reserved attributes</a>. This means that you lose Datadog telemetry and the benefits of <a href="https://docs.datadoghq.com/getting_started/tagging/unified_service_tagging/?tab=kubernetes">unified service tagging</a>. For example, say your syslogs are sent to <code>datadog_archives</code> and those logs have the status tagged as <code>severity</code> instead of the reserved attribute of <code>status</code> and the host tagged as <code>hostname</code> instead of the reserved attribute <code>hostname</code>. When these logs are rehydrated in Datadog, the <code>status</code> for the logs are all set to <code>info</code> and none of the logs will have a hostname tag.</div>
```

Comment:
let's move this right above the "Configuration File" header

Reply:
updated!